Automattic/jetpack
# CRM: Adding Automation Contact Actions

## Proposed changes:

* This PR adds Automation Contact actions - just the 'update contact' action and 'delete contact' action)
* This PR also includes tests

### Other information:


## Jetpack product discussion

https://github.com/Automattic/zero-bs-crm/issues/3125

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:


* Run tests by navigating to the CRM directory in your local installation after running `jetpack build plugins/crm` with this PR applied, and then run `composer phpunit -- --testdox --testsuite automation` (note this will currently run all Automation tests)
* To test the 'Update_Contact' class or the 'Delete_Contact' class, in both respective tests you can change the expected value in `$contact_data_expected` to be something else, and the tests should fail, showing the expected value ('Customer').
